src/base/bittorrent/magneturi.h

```diff
@@ -49,7 +50,7 @@ namespace BitTorrent
         bool isValid() const;
         InfoHash hash() const;
         QString name() const;
-        QList<TrackerEntry> trackers() const;
+        QVector<TrackerEntry> trackers() const;
         QList<QUrl> urlSeeds() const;
```

Comment:
maybe later, it hasn't bother me enough yet.

Reply:
Sorry, I don't quite understand your idea... having two identical cases next to each other, you fix only one of them, and the second "hasn't bother" you.
In any case, I will not reject it, because it looks good even in such an "incomplete" form.

Reply:
>and the second "hasn't bother" you.

urlSeeds is not used as often as tracker, this is in my case.